tests/use-cases/test_mnist.py

```diff
@@ -87,7 +87,8 @@ def test_mnist_inference_torch(torch_env, install_requirements):
     )
     with tempfile.TemporaryDirectory() as temp_dir:
         # Create fake inference dataset and checkpoint
-        generate_model_cmd = f"{torch_env}/bin/python {exec} " f"--root {temp_dir}"
+        generate_model_cmd = (f"{torch_env}/bin/python {exec} "
```

Comment:
Generally, when we have to use multiple lines for the contents of some parenthesis (for "(", "[" and "{"), we use newlines after the opening parenthesis and before the closing parenthesis. In this case, the code would thus be 

```python
generate_model_cmd = (
 ...
)
```